packages/headless/doc-parser/src/interface-resolver.ts

```diff
@@ -78,7 +74,7 @@ function isReference(m: ApiPropertySignature) {
 function buildEntityFromProperty(p: ApiPropertySignature) {
   return buildEntity({
     name: p.name,
-    desc: p.tsdocComment?.emitAsTsdoc() || '',
+    comment: (p.tsdocComment as unknown) as DocComment,
```

Comment:
The `tsdocComment` has type `DocComment` coming from `@microsoft/api-extractor-model` has one private property whose type is different from the `DocComment` type that comes from `@microsoft/tsdoc`